packages/components/src/tooltip/index.tsx

```diff
@@ -64,17 +87,30 @@ function Tooltip( props: TooltipProps ) {
 	}
 	computedPlacement = computedPlacement || 'bottom';
 
+	// Disable reason: the hook can not run conditionally.
+	// eslint-disable-next-line @wordpress/no-unused-vars-before-return
 	const tooltipStore = Ariakit.useTooltipStore( {
 		placement: computedPlacement,
 		showTimeout: delay,
 	} );
 
+	if ( isNestedInParentTooltip ) {
```

Comment:
Any potential core use cases with nested behavior that could be affected by this change?

Reply:
I can't think of any, if anything because, prior to this PR, I don't think that nesting `Tooltip` components would not work correctly.